gtsfm/densify/interface_patchmatchnet.py

```diff
@@ -0,0 +1,178 @@
+"""Interface class from gtsfmData to patchmatchnetData
+
+Authors: Ren Liu
+"""
+from typing import Any, Dict, List, Tuple
+
+import cv2
+import numpy as np
+from torch.utils.data import Dataset
+
+from gtsfm.common.image import Image
+from gtsfm.common.gtsfm_data import GtsfmData
+from gtsfm.densify.mvs_math import piecewise_gaussian
+
+NUM_PATCHMATCHNET_STAGES = 4
+
+
+class PatchmatchNetData(Dataset):
+    """PatchmatchNetData class for Patchmatch Net. It contains the interface from GtsfmData"""
+
```

Comment:
Does this class have an equivalent in the original codebase? Should we add a URL as reference?

Reply:
included at the beginning of the class now

Reply:
thanks